buildcontext/gitlookup.go

```diff
@@ -240,10 +240,7 @@ func (gl *GitLookup) detectProtocol(host string) (protocol string, err error) {
 var errNoRCHostEntry = fmt.Errorf("no netrc host entry")
 
 func (gl *GitLookup) lookupNetRCCredential(host string) (login, password string, err error) {
-	homeDir, _, err := cliutil.DetectHomeDir()
-	if err != nil {
-		return "", "", errors.Wrap(err, "failed to detect home dir")
-	}
+	homeDir, _ := cliutil.DetectHomeDir()
```

Comment:
This means that the fallback is `""` right?

I guess that's the best we can do if we tried everything.

Reply:
The fallback is `"/etc"`, which happens under https://github.com/earthly/earthly/pull/1027/files#diff-b8cb87e3ddc4c19120c5a344c081c9f68b80be59f98f1d1bf7b73ae791c4cbeaR104

This function should never return `""`.

Reply:
Ah right :+1: